src/components/Dashboard.vue

```diff
@@ -86,8 +86,8 @@ export default {
         firstName: names[0],
         lastName: names[1],
         middleName: '',
-        minor: [],
-        minorFN: []
```

Comment:
I don't believe these should be deleted

Reply:
I added it back but not sure if they have a use

Reply:
That's fine, I think I added them in the codebase before we actually implemented minors. Don't want to delete them in case that leads to any conflicts though.